Tester/Runner/Output/JUnitWriter.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+/**
+ * This file is part of the Nette Tester.
+ * Copyright (c) 2009 David Grudl (http://davidgrudl.com)
+ */
+
+namespace Tester\Runner\Output;
+
+use Tester,
+	Tester\Dumper,
+	Tester\Runner\Runner;
+
+
+/**
+ * JUnit xml format writer.
+ */
+class JUnitWriter implements Tester\Runner\OutputHandler
+{
+	/** @var Runner */
+	private $runner;
+
+	/** @var resource */
+	private $file;
+
+	/** @var string */
+	private $buffer;
+
+	/** @var float */
+	private $time;
+
+	public function __construct(Runner $runner, $file = 'php://output')
+	{
+		$this->runner = $runner;
+		$this->file = fopen($file, 'w');
+	}
+
+
+	public function begin()
+	{
+		$this->time = -microtime(TRUE);
```

Comment:
Someone probably thought this was a nice hack. But I don't see the benefit either :)